scraper/scraper/unsw/TimetableScraper.ts

```diff
@@ -236,10 +236,10 @@ export class TimetableScraper {
       for (let term = 0; term < courses.length; ++term) {
         const course = courses[term];
         removeDuplicateStreams(course);
+        removeDuplicateTimes(course, term);
```

Comment:
I think it would be better to swap the order of these, since this may make more of the times duplicates which can then be removed by `removeDuplicateStreams`.

```suggestion
        removeDuplicateTimes(course, term);
        removeDuplicateStreams(course);
```